Gemfile.lock

```diff
@@ -151,7 +151,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
don't remove the enterprise sidekiq-related gems

Reply:
I am not sure why these changes to Dangerbot and Gemfile were needed as part of this PR anyway. 

`git checkout Gemfile.lock -- master`
`git checkout Jenkinsfile -- master`